editoast/osm_to_railjson/justfile

```
@@ -0,0 +1,30 @@
+#!/usr/bin/env just --justfile
+
+# Display available recipes
+help:
+    @just --list --unsorted
+
+# Run the application; does not start other services
+run *args:
+    cargo run {{args}}
+
+# Install required dependencies
+install:
+    cargo build
+
+# Run the tests
+test:
+    cargo test
```

nitpick: for coherence with other Rust projects.

```suggestion
    cargo test --workspace
```

---

We don’t want --workspace for osm_to_railjson: it would check editoast as a whole, and not just this module. So I think here it would be better not